ui/src/routes/admin/users/index.tsx

```diff
@@ -152,14 +152,13 @@ const Users = () => {
             <TooltipTrigger asChild>
               <Button asChild size='sm' className='ml-auto gap-1'>
                 <Link to='/admin/users/create-user'>
-                  Add user
+                  Create new user
```

Comment:
As "create" somewhat implies "new", I'd just say "create user" here for brevity.